content/en/docs/ops/ambient/usage/ztunnel/index.md

```diff
@@ -37,14 +37,12 @@ Some use cases of Istio in ambient mode may be addressed solely via the L4 secur
 
 ## Current Caveats {#caveats}
 
-Ztunnel proxies are automatically installed when one of the supported installation methods is used to install Istio ambient mesh. The minimum Istio version required for Istio ambient mode is `1.18.0`. In general Istio in ambient mode supports the existing Istio APIs that are supported in sidecar proxy mode. Since the ambient functionality is currently at an alpha release level, the following is a list of feature restrictions or caveats in the current release of Istio's ambient functionality (as of the `1.19.0` release). These restrictions are expected to be addressed/removed in future software releases as ambient graduates to beta and eventually General Availability.
+Ztunnel proxies are automatically installed when one of the supported installation methods is used to install Istio ambient mesh. The minimum Istio version required for Istio ambient mode is `1.21.0` (note: although Istio Ambient is functional since version 1.18.0, it is strongly recommended at this time to use version 1.21.0 or newer). In general Istio in ambient mode supports the existing Istio APIs that are supported in sidecar proxy mode. Since the ambient functionality is currently at an alpha release level, the following is a list of feature restrictions or caveats in the current release of Istio's ambient functionality (as of the `1.21.0` release). These restrictions are expected to be addressed/removed in future software releases as ambient graduates to beta and eventually General Availability. Note that if a version earlier than 1.21.0 is used, there will be additional restrictions not listed here. 
```

Comment:
Just use the alpha warning boilerplate here.

```suggestion
{{< boilerplate ambient-alpha-warning >}}

Since the ambient functionality is currently at an alpha release level, the following is a list of feature restrictions or caveats in ambient alpha. These restrictions are expected to be addressed/removed in future software releases as ambient graduates to beta and eventually General Availability. 
```